models/staging/sources.yml

```diff
@@ -1,300 +0,0 @@
-version: 2
-
-sources:
-  - name: dbt_artifacts
-    database: "{% if target.type not in ('spark', 'databricks') %}{{ var('dbt_artifacts_database', target.database) }}{% endif %}"
-    schema: "{{ var('dbt_artifacts_schema', target.schema) }}"
-    tables:
-      - name: model_executions
```

Comment:
For ease of upgrade, I think we should keep the source model names the same as the ones listed are (i.e. remove the `src_dbt__` prefixes). What do you reckon

Reply:
Per my message in slack I think this is good since it will make migration paths smoother.